examples/userapi/07_functions_on_subdomains.ipynb

```diff
@@ -0,0 +1,3028 @@
+{
```

Comment:
<p>what hardware are you using and NUMA detection failed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Mac M3</p>